packages/web-components/src/components/va-telephone/test/va-telephone.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import { VaTelephone } from '../va-telephone';
+
+describe('formatPhoneNumber', () => {
```

Comment:
These unit tests are because I wanted to test the static helper functions. I left them as static functions rather than pulling them out into a separate file because:

- They aren't used anywhere else
- They're pretty specific to each other/the telephone component
    - i.e., expecting `ext` in the string for creating an aria label.

I also made them static just because I would rather pass in explicit arguments instead of relying on class properties.